Modules/Loadable/Markups/MRML/vtkCurveMeasurementsCalculator.cxx

```diff
@@ -437,16 +437,19 @@ bool vtkCurveMeasurementsCalculator::CalculatePolyDataTorsion(vtkPolyData* polyD
     {
     binormal = binormals->GetTuple3(linePoints->GetId(idx+1));
     binormalNorm = sqrt(binormal[0]*binormal[0] + binormal[1]*binormal[1] + binormal[2]*binormal[2]);
-
-    normBinormal[0] = binormal[0] / binormalNorm;
-    normBinormal[1] = binormal[1] / binormalNorm;
-    normBinormal[2] = binormal[2] / binormalNorm;
-
-    // Local torsion
-    torsion = sqrt( (normBinormal[0]-prevNormBinormal[0])*(normBinormal[0]-prevNormBinormal[0])
-                  + (normBinormal[1]-prevNormBinormal[1])*(normBinormal[1]-prevNormBinormal[1])
-                  + (normBinormal[2]-prevNormBinormal[2])*(normBinormal[2]-prevNormBinormal[2]) )
-              / binormalNorm;
+    torsion = 0.0;
+    if (binormalNorm > 0.0)
```

Comment:
For correctness, should the computation of `prevNormBinormal` above be also revisited ?

Reply:
Good point, I'll add a check.